components/slack/slack.app.js

```diff
@@ -223,6 +244,46 @@ module.exports = {
       description: "Text of the message to send (see Slack's [formatting docs](https://api.slack.com/reference/surfaces/formatting)). This field is usually required, unless you're providing only attachments instead. Provide no more than 40,000 characters or risk truncation.",
       //example: "Hello world"
     },
+    name: {
+      type: "string",
+      label: "Name",
+      description: "Name of a single key to set.",
+      //example: "Hello world"
```

Comment:
I see that these were included in other props from prior commits. If you don't mind removing these commented-out lines, that would be a huge help.